app/views/question_flow/forms/_employment_status.html.slim

```diff
@@ -1,17 +1,23 @@
 - partner ||= false
 = form_for(@form, url: request.path, method: :put) do |form|
-  = render "shared/heading",
-            header_text: t("question_flow.#{i18n_key}.heading"),
-            tag: (:partner if partner)
+  = render "shared/heading", tag: (:partner if partner)
```

Comment:
ok - this is just us tidying up how we populate the headings (nothing to do with radio buttons).

Reply:
Yes, this is standardising - normally for 1-question pages we set the header as the legend of the form element, but we hadn't done that here.